Topic 1/src/point6/AbstractPayment.java

```diff
@@ -0,0 +1,46 @@
+package point6;
+
+public abstract class AbstractPayment {
+	private double amount;
+	private double discount;
+	private double percentage; 
+	
+	public AbstractPayment(double amount){
+		this.amount= amount;
```

Comment:
[High] It is not necessary to have an amount for payment creation. Even you don't need to store that information. You should have a *method* that receive the amount to pay.